Automattic/wp-calypso
# Happychat: Middleware refactor of HAPPYCHAT_SEND_BROWSER_INFO

Creates new action type `HAPPYCHAT_SEND_BROWSER_INFO` to move side-effects into middleware. Part of an ongoing refactor.

`HAPPYCHAT_SEND_BROWSER_INFO` gathers the site the user flagged as needing help with, and relevant screen/browser info, and sends it as an `info` message to the operator.

### To test

- Pull up the Happychat HUD and http://calypso.localhost:3000/help/contact on a paid user account
- Fill out the Help form and click the "Happychat" message.
- The HUD should receive the same browser info message it usually does, including accurate measurements and the correct site URL.
